client/src/cb-admin/pages/VisitsData.js

```diff
@@ -109,35 +105,47 @@ export default class VisitsDataPage extends React.Component {
         this.props.updateAdminToken(resVisitors.headers.authorization);
 
         const visits = resVisitors.data.result;
-        const [visitsNumbers, genderNumbers, activitiesNumbers, ageGroups, activities] = resStats.data.result; //eslint-disable-line
+
+        const [
+          visitsNumbers,
+          genderNumbers,
+          activitiesNumbers,
+          ageGroups,
+          activities,
+        ] = resStats.data.result; //eslint-disable-line
 
         this.setState({
-          visits: visitsNumbers,
           visitNumbers: this.getVisitsWeek(visitsNumbers),
           genderNumbers: this.getGendersForChart(genderNumbers),
           activitiesGroups: this.getActivitiesForChart(activitiesNumbers),
           ageGroups: this.getAgeGroupsForChart(ageGroups),
           activities: activities.map(activity => activity.name),
           visitsList: visits,
-          filteredVisitsList: visits,
         });
       })
-      .catch(() => this.setState({ errors: { general: 'Unknown error' } }));
+      .catch((error) => {
+        if (ErrorUtils.errorStatusEquals(error, 401)) {
+          this.props.history.push('/admin/login');
+        } else if (ErrorUtils.errorStatusEquals(error, 500)) {
+          this.props.history.push('/internalServerError');
+        } else {
+          this.setState({ errors: { general: 'Could not update activity' } });
```

Comment:
What does this error message have to do with what's actually going on? We're fetching visits data, not updating anything.